src/sentry/api/endpoints/project_group_index.py

```diff
@@ -270,7 +270,12 @@ def delete(self, request: Request, project) -> Response:
         any data mutation.
 
         :qparam int id: a list of IDs of the issues to be removed.  This
-                        parameter shall be repeated for each issue.
+                        parameter shall be repeated for each issue, e.g.
+                        `?id=1&id=2&id=3`. If this parameter is not provided,
+                        it will attempt to remove the first 1000 issues.
+        :qparam querystring query: an optional Sentry structured search
```

Comment:
If you look at the bulk delete tests you will see that it is being used.
You can see this documented in the get method:
https://github.com/getsentry/sentry/blob/90006fc294c1160979416ca11cda2c75846af8b1/src/sentry/api/endpoints/project_group_index.py#L75-L77

It is likely that more options can be passed and documented, however, I will leave that for another pull request.